Homework - 02 Basic Python.ipynb

```
@@ -29,11 +64,81 @@
+    "user.append(random.randrange(1,11))\n",
+    "print(user)\n",
+    "\n",
+    "while True:\n",
+    "    game_over = input('do you want another card, yes or no: ')\n",
+    "    if game_over == \"no\":\n",
+    "        if sum(user) > sum(computer):\n",
+    "            print(\"computer hand: \" + str(computer))\n",
+    "            print(\"user hand: \" + str(user))\n",
+    "            print (\"game over. user wins!\")\n",
+    "            break\n",
+    "        if sum(user) < sum(computer):\n",
+    "            print(\"computer hand: \" + str(computer))\n",
+    "            print(\"user hand: \" + str(user))\n",
+    "            print(\"game over. computer wins!\")\n",
+    "            break\n",
+    "    \n",
+    "    if game_over == \"yes\":\n",
+    "        computer.append(random.randrange(1,11))\n",
+    "        user.append(random.randrange(1,11))\n",
+    "        print(user)\n",
+    "        \n",
+    "        if sum(computer) == 21:\n",
+    "            print(\"computer hand: \" + str(computer))\n",
+    "            print(\"user hand: \" + str(user))\n",
+    "            print(\"21! computer wins!\")\n",
+    "            break\n",
+    "        elif sum(user) == 21:\n",
+    "            print(\"computer hand: \" + str(computer))\n",
+    "            print(\"user hand: \" + str(user))\n",
+    "            print(\"21! you win!\")\n",
+    "            break\n",
+    "        elif sum(user) > 21:\n",
+    "            print(\"computer hand: \" + str(computer))\n",
+    "            print(\"user hand: \" + str(user))\n",
+    "            print(\"you lose ):\")\n",
+    "            break\n",
+    "        elif sum(computer) > 21:\n",
+    "            print(\"computer hand: \" + str(computer))\n",
+    "            print(\"user hand: \" + str(user))\n",
+    "            print(\"computer over 21. you win by default!\")\n",
+    "            break\n",
+    "        elif where_user < 21:\n",
```

what is `where_user`? that doesn't seem to be defined anywhere? Python should throw an error here.